frameworks/g-cloud-9/manifests/edit_submission.yml

```diff
@@ -19,70 +19,73 @@
     - serviceCategoriesSoftware
     - serviceCategoriesHosting
     - serviceCategoriesSupport
-    - serviceFeaturesAndBenefits
+-
+  name: Service features and benefits
+  editable: false
+  edit_questions: true
+  questions:
+    - multiqServiceFeaturesAndBenefits
 -
   name: Service scope
   editable: false
   edit_questions: true
   questions:
+    - multiqServiceAddOn
+    - cloudDeploymentModel
     - serviceConstraints
-    - buyerResponsibilities
+    - systemRequirements
 -
   name: Reselling
   editable: false
   edit_questions: true
   questions:
-    - reselling
+    - supplierType
```

Comment:
I've tried to keep the filenames the same as they name/headings of the questions so it's easier to find the file from looking at the content on the page. This feels more valuable to me but happy to discuss if you think otherwise.